README.md

```diff
@@ -1,679 +1,895 @@
+
+---
     
-    For more details: [batch monitoring settings](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#datarobot.models.BatchMonitoringJob.run) see the DataRobot docs.
+#### `UpdateMonitoringSettingsOperator`
+
+Updates monitoring settings for a deployment.
+
+Parameters:
+
+| Parameter                   | Type | Description |
+|-----------------------------|------|-------------|
+| `deployment_id`             | str  | The DataRobot deployment ID. |
+
+Sample config params:
+
+```
+"target_drift_enabled": True,
+"feature_drift_enabled": True,
+"association_id_column": ["id"],
+"required_association_id": False,
+"predictions_data_collection_enabled": False,
+```
+
+---
+
+#### `BatchMonitoringOperator`
+
+Creates a batch monitoring job for the deployment.
+
+Returns a batch monitoring job ID.
+
+Prerequisites:
+
+- Use `GetOrCreateCredentialOperator` to pass a `credential_id` from the preconfigured DataRobot Credentials (Airflow Connections) or manually set the `credential_id` parameter in the config.
+
+    > **Note:** You can [add S3 credentials to DataRobot via the Python API client](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/reference/admin/credentials.html#s3-credentials).
+
+- _Or_ use a Dataset ID from the DataRobot AI Catalog.
+- _Or_ use a DataStore ID for a JDBC source connection; you can use `GetOrCreateDataStoreOperator` to pass `datastore_id` from a preconfigured Airflow Connection.
+
+Parameters:
+
+| Parameter                   | Type | Description |
+|-----------------------------|------|-------------|
+| `deployment_id`             | str  | The DataRobot deployment ID. |
+| `datastore_id`              | str  | The DataRobot datastore ID. |
+| `credential_id`             | str  | The DataRobot credentials ID. |
+
+Sample config params:
+
+<details>
+
+<summary>Sample config</summary>
+
+``` json
+"deployment_id": "61150a2fadb5586af4118980",
+"monitoring_settings": {
+    "intake_settings": {
+        "type": "bigquery",
+        "dataset": "integration_example_demo",
+        "table": "actuals_demo",
+        "bucket": "datarobot_demo_airflow",
+    },
+    "monitoring_columns": {
+        "predictions_columns": [
+            {"class_name": "True", "column_name": "target_True_PREDICTION"},
+            {"class_name": "False", "column_name": "target_False_PREDICTION"},
+        ],
+        "association_id_column": "id",
+        "actuals_value_column": "ACTUAL",
+    },
+}
+```
+
+</details>
+
+<details>
+
+<summary>Sample config: Manually set credential ID</summary>
+
+``` json
+"deployment_id": "61150a2fadb5586af4118980",
+"monitoring_settings": {
+    "intake_settings": {
+        "type": "bigquery",
+        "dataset": "integration_example_demo",
+        "table": "actuals_demo",
+        "bucket": "datarobot_demo_airflow",
+        "credential_id": "63eb7dfce1274472579f6e1c"
```

Comment:
hide this credential id

Reply:
I think it is ok to display an example `id` right? There's no secret information in the ID... it would be nice if it was a made-up value as well just to be safe.

Reply:
I just added `<credential_id>` instead of a value. I'm not sure if these were made up. Let me know if we should replace the placeholder with a fake value.